lib/cdo/cloud_formation/cdo_app.rb

```diff
@@ -75,7 +75,7 @@ def initialize(**options)
 
       log_resource_filter.push 'FrontendLaunchConfig', 'ASGCount'
       tags.push(key: 'environment', value: rack_env)
-      tags.push(key: 'owner', value: Aws::STS::Client.new.get_caller_identity.arn) if rack_env?(:adhoc)
+      tags.push(key: 'owner', value: Aws::STS::Client.new.get_caller_identity.arn.split(':').last) if rack_env?(:adhoc)
```

Comment:
It’s maybe more verbose, but we could use the ARN Parser:

`aws::ARNParser.parse(Aws::STS::Client.new.get_caller_identity.arn).resource`

Reply:
I like that better

Reply:
```suggestion
      tags.push(key: 'owner', value: Aws::ARNParser.parse(Aws::STS::Client.new.get_caller_identity.arn).resource) if rack_env?(:adhoc)
```